### ISSUE TITLE
Only converts first chapter of DVD

### ISSUE THREAD

**Issue by usemoslinux** [danielgtaylor/arista#56]
I'm trying to convert a DVD to DivX. Conversion starts fine, but suddenly stops after finishing the first chapter (30 secs. aprox).

I'm using Ubuntu, with ubuntu-restricted-extras installed. I've also executed the sudo /usr/share/doc/libdvdread4/install-css.sh script to be able to play and read DVDs. Before running this script, Arista seemed to start the conversion but nothing happened.

**Comment by danielgtaylor** [comment]
Arista scans the DVD and tries to find the longest chapter, then rips it. Currently there's no easy way to figure out what chapter titles or actual durations are, so sometimes it makes the wrong choice. I'm waiting on GStreamer to provide me with more information so that I can allow users to choose titles, subtitles, and audio tracks, but until then there is very little I can do.

**Comment by danielgtaylor** [comment]
Duplicate of issue #42
